docs/reference/snapshot-restore/apis/create-snapshot-api.asciidoc

```diff
@@ -0,0 +1,122 @@
+[[create-snapshot-api]]
+=== Create index snapshot API
+++++
+<titleabbrev>Create index snapshot</titleabbrev>
+++++
```

Comment:
By default, snapshots include an entire cluster, including the cluster state.
With #57675, snapshots now support data streams in `master` and `7.x`.

I'd avoid associating snapshots too closely with indices.

```suggestion
[[create-snapshot-api]]
=== Create snapshot API
++++
<titleabbrev>Create snapshot</titleabbrev>
++++
```